packages/twenty-server/src/engine/core-modules/auth/guards/google-apis-oauth-exchange-code-for-token.guard.ts

```diff
@@ -41,9 +41,9 @@ export class GoogleAPIsOauthExchangeCodeForTokenGuard extends AuthGuard(
       !this.environmentService.get('MESSAGING_PROVIDER_GMAIL_ENABLED') &&
       !this.environmentService.get('CALENDAR_PROVIDER_GOOGLE_ENABLED')
     ) {
-      throw new AuthException(
+      throw new EnvironmentException(
         'Google apis auth is not enabled',
-        AuthExceptionCode.FORBIDDEN_EXCEPTION,
+        EnvironmentExceptionCode.ENVIRONMENT_VARIABLES_NOT_FOUND,
```

Comment:
As discussed offline, I'm not sure we want to throw EnvironmentException outside of the environment service. Not a blocker yet though

Reply:
see #8845